README.md

```diff
@@ -2,60 +2,22 @@
 
 This is a temporary repo designed to give an easy-to-understand deployment of the apache / haproxy / shiny stack; it does not include configurable applications or anything like that. See it online at https://shiny-dev.dide.ic.ac.uk (DIDE network only).
 
-## The components
 
-Some of these are docker images, and they are not set to pull, so you will want to rebuild. All build very quickly.
+## Running in Kubernetes
 
-### apache
+The following is guide to run the shiny server in kubernetes. They are based on running a Kind cluster. The configuration may
+nee to be adjusted for other k8s clusters.
 
-Running an unmodified httpd container (previously was 2.4, we'll update once we know this works). The configuration ([`apache/httpd.conf`](httpd/httpd.conf)) and certificates (`apache/ssl`) will be read-only mounted into the container. You need to fetch the ssl key and certificate, run `./apache/configure_ssl` to do this (only needs to be done if they change or if the `ssl` directory is deleted)
+### Prerequisites
 
-### haproxy
+A production kubernetes cluster using k3s is needed to be setup first. To setup a k8s cluster follow the guide [here](https://mrc-ide.myjetbrains.com/youtrack/articles/RESIDE-A-31/Setting-up-Kubernetes-k8s-Cluster). Note: If using dev cluster (KIND), the storageClassName: local-path & longhorn is unavailable and you will need to provision own storage class and persistant volume. 
```

Comment:
> storageClassName: local-path & longhorn is unavailable and you will need to provision own storage class and persistant volume.

Sorry, don't really understand this. Why aren't these available? You talk about using `local-path` below, in persistance.yaml.

Reply:
have updated to avoid confusion